roles/node-images-application/tasks/main.yml

```diff
@@ -37,6 +57,7 @@
       tags: google
   tags:
     - google
+    - never
```

Comment:
Curiosity question, why did we add the `never` tag?

Reply:
Solves the same thing the common tag solved, more or less.

Reply:
One of the Packer provisioners calls the playbook without tags (which ignores tags and runs everything). This is a way of saying, "run only if **--tags google** is present. "always" and "never" are built-in ansible tags.